tasks/README.md

```diff
@@ -347,3 +353,14 @@ Some examples:
 The JSON schema for errors is included in [error.json](error.json).
 
 [Bolt]: https://github.com/puppetlabs/bolt
+
+## Exectuting Remote Tasks
+
+Tasks may be written to execute on a proxy target and interact remotely with
+the specified target. This is useful when the target has a limited shell
+environment or only exposes an HTTP API.
+
+- The task runner must refuse to execute tasks implementations that do not have `remote` set on remote targets.
+- The task runner must refuse to execute task implementations that do have `remote` set on normal targets.
```

Comment:
An implementation that changes behaviour depending on the presence of the `_target` metaparam is easily imaginable. Are we making this too strict by disallowing a task to provide a single implementation for both local and remote execution?

Reply:
This is accomplished but having two entries in `implementations` that point to same file, one remote the other not.

Reply:
ah, oh. that makes sense.